backend/app/services/knowledge/knowledge_service.py

```
@@ -1704,16 +1682,49 @@ def get_all_knowledge_bases_grouped(
 
+        shared_namespace_names = {
+            kb.namespace for kb in shared_kbs if kb.namespace != "default"
+        }
+        shared_namespaces = {}
+        if shared_namespace_names:
+            shared_namespaces = {
+                ns.name: ns
+                for ns in db.query(Namespace)
+                .filter(
+                    Namespace.name.in_(shared_namespace_names),
+                    Namespace.is_active == True,
```

Ruff (0.15.7)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)